tests/test_bot.py

```diff
@@ -25,12 +25,11 @@
 
 
 @pytest.mark.skipif(not GithubUtils.github_api_status(), reason="Github api is down")
-@pytest.mark.skipif(not os.environ.get('GITHUB_TOKEN') and not os.environ.get('GITHUB_USER'),
-                    reason="missing GITHUB_TOKEN and GITHUB_USER variables")
+@pytest.mark.skipif(not os.environ.get('GITHUB_TOKEN'),
+                    reason="missing GITHUB_TOKEN environment variable")
 class TestBot:
     """ Tests parts of bot workflow"""
     github_token = os.environ.get('GITHUB_TOKEN')
-    github_user = os.environ.get('GITHUB_USER')
```

Comment:
There are 2 variables specified in [.travis.yml](https://github.com/user-cont/release-bot/blob/master/.travis.yml#L12). Can one of them be removed?

Reply:
Yes, but the thing is that we don't know which one b/c we can't decrypt that - travis doesn't hand out those private keys. I think it would be better to define those secure things via web ui.